include/re/re.h

```diff
@@ -85,7 +85,8 @@ enum re_errno {
 	RE_EFLAG        = 11 | RE_MARK,
 	RE_EXCLOSEFLAGS = 12 | RE_MARK,
 	RE_EXUNSUPPORTD = 13 | RE_MARK,
-	RE_EBADCP       = 14 | RE_MARK
+	RE_EBADCP       = 14 | RE_MARK,
+	RE_BADCOMMENT   = 15 | RE_MARK
```

Comment:
amazing that a regexp dialect has a notion of comments, let alone ill-formed ones

Reply:
I actually like that it does.  People can write long, complex regexps, and comments along with the PCRE/Perl extended syntax makes them a lot more readable and maintainable.